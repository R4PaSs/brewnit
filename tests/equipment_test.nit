import dummy_db

with ctx = new DBContext do
	var res = ctx.equipment_worker.fetch_multiple(ctx, "* FROM equipments")
	for i in res do print i
end
