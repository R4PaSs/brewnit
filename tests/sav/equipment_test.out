Equipment: Home, Mean efficiency: 0.65%, Total volume: 30.0 L, trub losses: 2.0 L, loss/hour: 5.0%
